subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/bintray/BintrayReleasePlugin.java

```diff
@@ -61,7 +64,12 @@ public void execute(JavaBintrayPlugin plugin) {
                         deferredConfiguration(subproject, new Runnable() {
                             public void run() {
                                 UpdateReleaseNotesTask updateNotes = (UpdateReleaseNotesTask) project.getTasks().getByName(ReleaseNotesPlugin.UPDATE_NOTES_TASK);
-                                updateNotes.setPublicationRepository(BintrayUtil.getRepoLink(bintray));
+                                String userSpecifiedRepo = conf.getLenient().getReleaseNotes().getPublicationRepository();
+                                if (userSpecifiedRepo != null) {
+                                    updateNotes.setPublicationRepository(userSpecifiedRepo);
+                                } else {
```

Comment:
Unit test please :) Let us know if you need help.

Reply:
A little bit help would be great 👍 
I've never used spock before. I've also never tested a gradle plugin 🙃 
I've tried it with something like that:
```
    def "applies"() {
        expect:
        project.plugins.apply(BintrayReleasePlugin)
        project.plugins.apply(JavaBintrayPlugin)
        //def repository = project.tasks.getByName("updateReleaseNotes").getPublicationRepository()
        //assert repository == null
        project.evaluate()
        def repositoryNonNull = project.tasks.getByName("updateReleaseNotes").getPublicationRepository()
        assert repositoryNonNull != null
    }
```
But unfortunately that isn't working..